CONTRIBUTING.md

```diff
@@ -247,6 +247,10 @@ npm run samples:serve
 
 Head to [https://localhost:8000/](https://localhost:8000/) to use the samples
 
+> NOTE: The samples use a self-signed certificate.
+> Some browsers such as Google Chrome may not allow access by default for security reasons.
```

Comment:
```suggestion
> Some browsers such as Google Chrome/Edge may not allow access by default for security reasons.
```

Reply:
Made it a bit more genetic and said "chromium-based browser"